scripts/generate-testcases.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+import sys
+import re
+from jinja2 import Template
```

Comment:
I assume this is installed on Permian runner hosts, as part of a still open PR for builders? There is so much that floats around our infra that it might make sense to comment that this bit is installed on the runners. I think it's an unusual place to have a dependency.

To see where this comes from, I checked the dockerfiles, install_dependencies, spec file, then realized this is the kickstart tests repo so checked other dockerfiles, and finally grepped half the disk... and didn't find any reference!

Reply:
I guess it is indirect dependency of Permian (https://gitlab.cee.redhat.com/rhinstaller/builders/-/merge_requests/102), maybe even the original kickstart tests runner - I was able to import it on our regular kstest runner.